hamilton/io/utils.py

```diff
@@ -17,3 +24,28 @@ def get_file_metadata(path: str) -> Dict[str, Any]:
         "last_modified": os.path.getmtime(path),
         "timestamp": datetime.now().utcnow().timestamp(),
     }
+
+
+def get_sql_metadata(query_or_table: str, results: Union[int, pd.DataFrame]) -> Dict[str, Any]:
```

Comment:
can you add unit tests on this please? This will help document the expected behavior.

Reply:
i have unit tests on the get sql metadata function! unless you meant the get file metadata function?

Reply:
oh nvm! I see it. Must of missed it?